src/com/lftechnology/java/training/srijan/basics/coreapp/controller/LoginController.java

```diff
@@ -0,0 +1,41 @@
+
+package com.lftechnology.java.training.srijan.basics.coreapp.controller;
+
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.Employee;
+import com.lftechnology.java.training.srijan.basics.coreapp.domain.UserRole;
+import com.lftechnology.java.training.srijan.basics.coreapp.service.EmployeeServiceImpl;
+import com.lftechnology.java.training.srijan.basics.coreapp.service.ServiceFactory;
+import com.lftechnology.java.training.srijan.basics.coreapp.ui.EmployeeView;
+
+import java.util.Scanner;
+
+public class LoginController {
+
+	UserRole role;
+
+	public LoginController() {
+		this.role = UserRole.NORMAL;
+	}
+	EmployeeServiceImpl employeeServiceImpl =
+		ServiceFactory.getEmployeeService();
+	Employee e = new Employee();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)